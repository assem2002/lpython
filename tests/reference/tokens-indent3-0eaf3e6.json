{
    "basename": "tokens-indent3-0eaf3e6",
    "cmd": "lpython --no-color --show-tokens {infile} -o {outfile}",
    "infile": "tests/tokens/errors/indent3.py",
    "infile_hash": "cd9102f8d110fa0d6b12c07025d6646de807b60ee17b4604d9df08dc",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "tokens-indent3-0eaf3e6.stderr",
    "stderr_hash": "6a6419de94ff032184aedcbe49c8243becdef462d84e66cabd891467",
    "returncode": 1
}