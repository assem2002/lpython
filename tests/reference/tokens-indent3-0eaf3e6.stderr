tokenizer error: Expected an indented block.
 --> tests/tokens/errors/indent3.py:5:1
  |
5 |     pass
  | ^^^^ 


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
